packages/logs/README.md

```diff
@@ -309,8 +309,9 @@ You can update the following event properties:
 | `message`       | String | The content of the log.                                                                          |
 | `error.stack`   | String | The stack trace or complementary information about the error.                                    |
 | `http.url`      | String | The HTTP URL.                                                                                    |
+| `context`       | String | Extra contextual attributes added with the logger.                                               |
 
-**Note**: The Browser SDK will ignore modifications made to event properties not listed above. For more information about event properties, see the [Browser SDK repository][5].
+**Note**: The Browser SDK will ignore modifications made to event properties not listed above. For more information about event properties, see the [Browser SDK repository][6].
```

Comment:
Actually, since logs customers can already overwrite any property, we decided to allow the modifications of all properties with https://github.com/DataDog/browser-sdk/pull/909

> Allow modification of any field, not only sensitive fields.
> Allow addition of new fields.

Reply:
```suggestion
```

Reply:
^ Suggestion to remove the note following @bcaudan's input.